### DISCUSSION TITLE
Ugs 219 fluidnc problem.

### DISCUSSION THREAD

**Discussion by Monberg75** [winder/Universal-G-Code-Sender#2621] [Help]
I'm experiencing that ugs 219 is moving center on my machine.

I do as I usely do. Find center. Zero all axis, but when I start the project it seems to move the center a bit to the left.

Tried the same files twice on 219, switched back to 217 and all was good.

I don't understand it.

**Comment by breiler** [comment]
Are you running a gcode file or a design file from the designer?

**Comment by Monberg75** [comment]
It's a gcode file from fusion.

fre. 18. okt. 2024 19.34 skrev Joacim Breiler ***@***.***>:
